reactor-netty-http/src/main/java/reactor/netty/http/client/HttpClient.java

```diff
@@ -515,7 +516,13 @@ public final HttpClient baseUrl(String baseUrl) {
 	 * @return a new {@link HttpClient}
 	 */
 	public final HttpClient compress(boolean compressionEnabled) {
+		configuration().headers.remove(HttpHeaderNames.ACCEPT_ENCODING);
```

Comment:
This will remove any custom `Accept-Encoding` header that was set prior this invocation.
Why do we need to do this? It will be better to preserve the custom headers.